app/views/pages/_form.html.erb

```diff
@@ -1,27 +1,38 @@
-<%= f.govuk_error_summary %>
-<!-- Long question text input -->
-<%= f.govuk_text_field :question_text, label: { size: 'm' } %>
+<%= form_with model: [form_object, page_object], url: action_path do |f| %>
+  <%= f.hidden_field :answer_type if is_new_page %>
 
-<hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible" />
+  <%= f.govuk_error_summary %>
 
-<!-- Hint text -->
-<%= f.govuk_text_field :hint_text, label: { size: 'm' } %>
+  <%= f.govuk_text_field :question_text, label: { size: 'm' } %>
 
-<hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible" />
+  <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible" />
 
-<%= f.govuk_check_boxes_fieldset :is_optional, multiple: false, legend: { text: t("pages.add_rules_legend") , size: 'm' } do %>
-  <%= f.govuk_check_box :is_optional, true,0, multiple: false,
-                        small: true,
-                        label: { text: t("pages.make_optional_text")  },
-                        hint: { text: t("pages.make_optional_hint") } %>
+  <%= f.govuk_text_field :hint_text, label: { size: 'm' } %>
 
+  <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible" />
+
+  <%= f.govuk_check_boxes_fieldset :is_optional, multiple: false, legend: { text: t("pages.add_rules_legend") , size: 'm' } do %>
+    <%= f.govuk_check_box :is_optional, true,0, multiple: false,
+                          small: true,
+                          label: { text: t("pages.make_optional_text")  },
+                          hint: { text: t("pages.make_optional_hint") } %>
+
+  <% end %>
+
+  <%= govuk_summary_list do |summary_list|
+    summary_list.row do |row|
+      row.key(text: "Answer type")
+      row.value(text: t("helpers.label.page.answer_type_options.names.#{page_object.answer_type}"))
+      row.action(text: "Change",
+                 href: change_answer_type_path,
+                 visually_hidden_text: t("helpers.label.page.answer_type_options.names.#{page_object.answer_type}")
```

Comment:
I think  we want the hidden text to include 'answer type' here - that's how it is in the prototype. Think the logic here is that someone using speech recognition is probably more likely to say 'Click change answer type text' or 'Click change answer type' than 'Click change text'.

```suggestion
                 visually_hidden_text: "Answer type" + t("helpers.label.page.answer_type_options.names.#{page_object.answer_type}")
```

(there's probably a nicer way of concatenating these strings than that)

Reply:
Good spot - after some messing around I couldn't find a better way of interpolating within an interpolation, so the `+` seems like the easiest way of doing it.